src/examples/doxygen/delay.cpp

```diff
@@ -0,0 +1,24 @@
+#include <rpp/rpp.hpp>
+
+#include <chrono>
+#include <iostream>
+
+/**
+ * \example delay.cpp
+ **/
+int main()
+{
+    //! [delay]
+    std::cout << std::this_thread::get_id() << std::endl;
+    rpp::source::just(1, 2, 3)
+            .delay(std::chrono::seconds{1}, rpp::schedulers::new_thread{})
+            .as_blocking()
+            .subscribe([](int v) { std::cout << "[" << std::this_thread::get_id() << "] : " << v << std::endl; });
```

Comment:
I think it is also good idea to show diff in time. For example "time since subscription" because `delay` it is more about delaying, not using scheduler